### ISSUE TITLE
XDG Base Directory Specification

### ISSUE THREAD

**Issue by cgtx** [stefanoborini/vai#130]
I noticed that using vai creates the file `~/.vaistate`.  I try hard to keep my `~` clean.  One solution would be to follow the [XDG Base Directory Specification](http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html).  There is some debate that the existing categories don't cover a "state" file, so Debian has proposed inclusion of a [STATE directory](https://wiki.debian.org/XDGBaseDirectorySpecification#state).

In a nutshell, I would love to see vai use use `${XDG_STATE_HOME}/vai/vaistate` (defaults to `~/.local/state/vai/vaistate` if unset).  If their is interest, I can submit a pull request.  I just don't want to start if there is zero chance of it being merged.
